iot/api/Program.cs

```diff
@@ -617,6 +633,32 @@ public static object GetDeviceConfigurations(string projectId, string cloudRegio
         }
         // [END iot_get_device_configurations]
 
+        // [START iot_get_device_states]
+        public static object GetDeviceStates(string projectId, string cloudRegion, string registryId, string deviceId)
+        {
+            var cloudIot = CreateAuthorizedClient();
+            // The resource name of the location associated with the key rings.
```

Comment:
nit: add an empty line between line 639 and the comment

Reply:
Done